src/router/index.ts

```diff
@@ -40,8 +41,27 @@ const loginGuard = (to: any, from: any, next: any) => {
 
 const routes: Array<RouteRecordRaw> = [
   {
-    path: "/",
-    redirect: "/tabs/search",
+    path: '/',
+    redirect: '/drafts'
+  },
+  {
+    path: '/drafts',
+    name: 'Drafts',
+    component: Drafts,
+    // beforeEnter: authGuard,
```

Comment:
Uncomment this changes as authentication is required here, just remote the permissionId